src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/CheckFormSteps.java

```diff
@@ -31,8 +34,17 @@ public class CheckFormSteps extends RemoteTestBase {
 	
 	private HomePage homePage;
 	
+	private String jsonData = "{\"resourceType\":\"Patient\",\"identifier\":[{\"id\":\"" + TestsUtil.generateRandomUUID() + "\",\"extension\":[{\"url\":\"http://fhir.openmrs.org/ext/patient/identifier#location\",\"valueReference\":{\"reference\":\"Location/0a2c0967-2a56-41c9-9ad5-0bd959861b42\",\"type\":\"Location\",\"display\":\"CS de la Croix-des-Bouquets\"}}],\"use\":\"usual\",\"type\":{\"text\":\"Code ST\"},\"system\":\"http://localhost:8000/openmrs/fhir2/6-code-st\",\"value\":\"" + TestsUtil.generateCodeST() + "\"}],\"active\":true,\"name\":[{\"id\":\"" + TestsUtil.generateRandomUUID() + "\",\"family\":\"Jenkins\",\"given\":[\"Josh\"]}],\"gender\":\"male\",\"birthDate\":\"1971-04-11\",\"deceasedBoolean\":false,\"address\":[{\"id\":\"" + TestsUtil.generateRandomUUID() + "\",\"extension\":[{\"url\":\"http://fhir.openmrs.org/ext/address\",\"extension\":[{\"url\":\"http://fhir.openmrs.org/ext/address#address1\",\"valueString\":\"Address17001\"}]}],\"use\":\"home\",\"city\":\"City7001\",\"state\":\"State7001\",\"postalCode\":\"47002\",\"country\":\"Country7001\"}]}";
+
+	private  String url = "https://iplus3.openelis-global.org/openmrs/ws/fhir2/R4/Patient/";
```

Comment:
@HerbertYiga , i think i missed this .
we dont hard code URLs , as this is some thing that changes.

can you just read the [base url ](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/src/test/resources/test.properties#L1)from the properties file ,and append whatever end point you want to use ?
Please clean all the existing PRs i already merged

Reply:
sure am gona do the clean up